test_root/uploads/.gitignore

```diff
@@ -1,4 +0,0 @@
-*.csv
```

Comment:
Instead of removing this file, you could keep it, but add an exception for the files you want to be able to checkin. Something like this:
```bash
*.csv
*.jpg
*.png
*.txt
!my_file.jpg
```